src/stcal/jump/jump.py

```diff
@@ -209,6 +210,15 @@ def detect_jumps(
     min_sat_radius_extend : float
         The minimum radius of the saturated core of a snowball for the core to
         be extended
+    minimum_groups: int
```

Comment:
```suggestion
    minimum_groups : int
```

Reply:
added the space here and in the suggestion below